otx/algorithms/classification/adapters/mmcls/nncf/registers.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from otx.algorithms.common.adapters.nncf.utils import is_nncf_enabled
+
+
+if is_nncf_enabled():
+    from nncf.torch import register_module
+    from timm.models.layers.conv2d_same import Conv2dSame
+
+    # Register custom modules.
+    # Users of nncf should manually check every custom
+    # layer with weights which should be compressed and
+    # in case such layers are not wrapping by nncf,
+    # wrap such custom module by yourself.
+    register_module(ignored_algorithms=[])(Conv2dSame)
```

Comment:
Could you explain a bit? `Conv2dSame` is used in our models but not wrapped by NNCF?

Reply:
Yes, exactly. NNCF only supports native pytorch modules. `Conv2dSame` module is used in a `EfficientNet-V2-S` classification model and this module is not wrapped by NNCF because it is defined by user, here `timm` library. This is a shortcomings of NNCF as one has to register every custom modules to be wrapped by NNCF. 
[Here](https://github.com/openvinotoolkit/nncf/blob/develop/docs/Usage.md#compression-of-custom-modules) is a relative section in NNCF docs